Source/FieldSolver/WarpXPushFieldsEM.cpp

```diff
@@ -41,6 +42,7 @@ WarpX::EvolveB (int lev, PatchType patch_type, amrex::Real a_dt)
     const int patch_level = (patch_type == PatchType::fine) ? lev : lev-1;
     const std::array<Real,3>& dx = WarpX::CellSize(patch_level);
     Real dtsdx = a_dt/dx[0], dtsdy = a_dt/dx[1], dtsdz = a_dt/dx[2];
+    Real dxinv = 1./dx[0];
```

Comment:
these constants should be constant :)
```suggestion
    const Real dxinv = 1./dx[0];
```